### ISSUE TITLE
Exception has occurred message looks out of place

### ISSUE THREAD

**Issue by Tyriar** [microsoft/vscode#151931]
Repro: Debug something with exceptions caught.

The text is small and bold and the `x` looks weird because it's larger than the content.

![image](https://user-images.githubusercontent.com/2193314/173342466-3f70bd6f-523c-4d81-9f94-3045bd34f2e5.png)

Compare to peek view which has the `x` right aligned

![image](https://user-images.githubusercontent.com/2193314/173342492-c0e55db5-cdb2-444d-866c-fded9d852736.png)

What an exception with a stack trace looks like:

![image](https://user-images.githubusercontent.com/2193314/173342723-07b3ff02-6dd1-4131-9642-b262fdb18a04.png)

**Comment by roblourens** [comment]
Looks centered for me but I see that it depends on editor font settings

![image](https://user-images.githubusercontent.com/323878/173421771-bab22736-08f3-4090-baf1-0c8c263038e2.png)

@misolori should it go to the right corner or stay next to the title?

**Comment by misolori** [comment]
is this using the peek widget? could we use the peek title that then places these close button in a consistent place?

**Comment by misolori** [comment]
can we style it like a peek widget?

![image](https://user-images.githubusercontent.com/35271042/173428117-9edaa3e3-89db-4c20-8b52-b594509eb1c2.png)

**Comment by roblourens** [comment]
I can even make it an actual peek widget that shares the same code, if you like that style. But then it looks the same as the F8 "go to next error" widget, is that an issue? The current styling is very loud which seems good for an exception

**Comment by Tyriar** [comment]
Some more options to help differentiate from problems (ignore the arrows on the right):

![image](https://user-images.githubusercontent.com/2193314/173582848-42c8d59d-a845-48b7-b4ec-c392c0ee3333.png)

![image](https://user-images.githubusercontent.com/2193314/173583112-4f0b1779-7e10-4720-afc2-229e622b7127.png)

![image](https://user-images.githubusercontent.com/2193314/173582925-64aeef66-9dbb-4a89-babb-15af2bec8a03.png)

Problem:

![image](https://user-images.githubusercontent.com/2193314/173582794-5a2c6fae-3500-4a52-add0-0f22028b71c9.png)

**Comment by misolori** [comment]
I don't think it would be an issue because it's still an error (though not the same type of error). If we are really worried about the confusion we could introduce a new icon + color set instead, but that might also equally confuse users. I'd say we try it out and see how it feels + watch the feedback (if there will be any).
